### PR TITLE
feat(server): simplified test running & IDE integrations

### PR DESCRIPTION
**Pull request by fabis94**
- Recommend mocha test runner & coverage extensions
- Added prettier to all packages, forgot to do it in previous PR (unrelated to this)
- Simplified test running scripts, moved most of the config from script arguments to config/env files
  -  One relatively big change is the introduction of `.env.test` which is loaded only when NODE_ENV=test and takes precedence over any values in `.env`
- Some random auto-reformatting

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,34 +37,34 @@ workflows:
...
-          command: "npm install"
-          working_directory: "packages/server"
-      - run: "dockerize -wait tcp://localhost:5432 -timeout 1m"
+          command: 'npm install'
+          working_directory: 'packages/server'
+      - run: 'dockerize -wait tcp://localhost:5432 -timeout 1m'
       - run:
-          command: "npm run test:report"
-          working_directory: "packages/server"
+          command: 'npm run test:coverage'
```
Comment (fabis94) [inline .circleci/config.yml:63]: only part I changed, everything else is autoformatting

File: packages/server/app.js [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
-const { startup: MatStartup } = require( `${appRoot}/logging/matomoHelper` )
-const prometheusClient = require( 'prom-client' )
-
-const { ApolloServer, ForbiddenError } = require( 'apollo-server-express' )
-
-require( 'dotenv' ).config( { path: `${appRoot}/.env` } )
-
-const { contextApiTokenHelper } = require( './modules/shared' )
-const knex = require( './db/knex' )
+require('./bootstrap')
```
Comment (fabis94) [inline packages/server/app.js:4]: only autoformatting

File: packages/server/modules/emails/services/verification.js [hunk 3]
```diff
@@ -1,42 +1,41 @@
-const appRoot = require( 'app-root-path' )
-require( 'dotenv' ).config( { 'path': `${appRoot}/.env` } )
-const crs = require( 'crypto-random-string' )
+require('../../../bootstrap')
```
Comment (fabis94) [inline packages/server/modules/emails/services/verification.js:1]: Moved common entrypoint init code to `bootstrap.js` for better DRYness

File: packages/server/modules/stats/tests/stats.spec.js [hunk 4]
```diff
@@ -1,89 +1,97 @@
...
-const { createPersonalAccessToken } = require( `${appRoot}/modules/core/services/tokens` )
-const { createStream } = require( `${appRoot}/modules/core/services/streams` )
-const { createObjects } = require( `${appRoot}/modules/core/services/objects` )
-const { createCommitByBranchName } = require( `${appRoot}/modules/core/services/commits` )
-
-const { beforeEachContext, initializeTestServer } = require( `${appRoot}/test/hooks` )
-const { createManyObjects } = require( `${appRoot}/test/helpers` )
-
-const { getStreamHistory, getCommitHistory, getObjectHistory, getUserHistory, getTotalStreamCount, getTotalCommitCount, getTotalObjectCount, getTotalUserCount } = require( '../services' )
+const expect = require('chai').expect
```
Comment (fabis94) [inline packages/server/modules/stats/tests/stats.spec.js:2]: should be only autoformat

File: packages/server/modules/emails/services/verification.js (hunk 3 above)
Comment (gjedlicska) [reply packages/server/modules/emails/services/verification.js:1]: why r we requiring the env file here?

Comment (gjedlicska) [reply packages/server/modules/emails/services/verification.js:1]: i think this is a leftover thing, u can just delete the bootstrap require all together

**Review by gjedlicska — APPROVED** [review]

**Review by gjedlicska — APPROVED** [review]

**Pull request merged**